src/strategies/math/README.md

```diff
@@ -6,21 +6,68 @@ Currently supported operations are:
 
 - `square-root`
 - `cube-root`
+- `min`
+- `max`
 
 The following example takes the cube root of a user's DAI token balance as voting score.
 
 ```json
 {
   "symbol": "MATH",
-  "strategy": {
-    "name": "erc20-balance-of",
-    "network": "1",
-    "params": {
-      "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
-      "symbol": "DAI",
-      "decimals": 18
+  "operands": [
+    {
+      "type": "strategy",
+      "strategy": {
+        "name": "erc20-balance-of",
+        "network": "1",
+        "params": {
+          "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
+          "symbol": "DAI",
+          "decimals": 18
+        }
+      }
     }
-  },
+  ],
```

Comment:
There should be a limit on how many operands can be passed in, else it would add additional load to the server

Reply:
This _is_ a limit on operand count. The number of operands is limited based on the operation used. `square-root` and `cube-root` take 1 operand, while `min` and `max` take 2. Passing in more operands result in an error.

That said, this strategy is similar to `multichain` in the sense that limiting the number of operands (strategies) on the strategy level doesn't really help, as the caller can simply nest more strategies inside, forcing the server to do an unbounded number of requests. The only way to truly mitigate the issue is to set the number of requests made on the caller's side.

Also, `multichain` does not impose a limit on the number of strategies either.